suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -1010,17 +1005,16 @@ export const getPendingAccount = ({
     };
 };
 
-export const getNetworkFeatures = ({
-    networkType,
+export const getNetworkAccountFeatures = ({
```

Comment:
renamed to reflect that it gets features either from `account`, or `network` if `accountType` is `'normal'`